cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -909,7 +900,7 @@ private void updateCluster(VertxTestContext context, Kafka originalAssembly, Kaf
         // create CM, Service, headless service, statefulset and so on
         ResourceOperatorSupplier supplier = ResourceUtils.supplierWithMocks(openShift);
         ClusterOperatorConfig config = ResourceUtils.dummyClusterOperatorConfig(VERSIONS);
-        CrdOperator mockKafkaOps = supplier.kafkaOperator;
+        var mockKafkaOps = supplier.kafkaOperator;
```

Comment:
Trying out @tombentley's suggestion of `var` instead of a full, and large, type signature, to remove warnings about raw types.